tour/basic-read-write.md

```diff
@@ -26,7 +26,8 @@ static void exercise(AccumuloClient client) throws Exception {
     // Read and print all rows of the "GothamPD" table. Try w/ resources will close for us.
     try (Scanner scan = client.createScanner("GothamPD", Authorizations.EMPTY)) {
         System.out.println("Gotham Police Department Persons of Interest:");
-        // A Scanner is an extension of java.lang.Iterable so behaves just like one.
+        
+        // Note: A Scanner is an extension of java.lang.Iterable so it will traverse through the table.
```

Comment:
```suggestion
        // Note: A Scanner is an extension of java.lang.Iterable so it will traverse through the scanner's range.
        // In this case, since no range was set on the scanner, it will traverse the entire table.
```